docs/windows.md

```diff
@@ -347,269 +324,55 @@ containerized), and you will need to run `Install-OVS.ps1` to install them.
+`<KubeletKubeconfigPath>` should be set by you. For example:
 
 ```powershell
-# kube-proxy mode is no longer supported starting with K8s version 1.26
-$InstallKubeProxy=$false
-$KubernetesVersion="v1.23.5"
+$KubernetesVersion="v1.29.0"
 $KubeConfig="C:/Users/Administrator/.kube/config" # admin kubeconfig
 $KubeletKubeconfigPath="C:/etc/kubernetes/kubelet.conf"
-if ($InstallKubeProxy) { $KubeProxyKubeconfigPath="C:/Users/Administrator/kubeproxy.conf" }
 ```
 
 ```powershell
-$TAG="v1.14.0"
+$TAG="v1.15.0"
 $KubernetesVersion="<KubernetesVersion>"
 $KubeConfig="<KubeconfigPath>"
 $KubeletKubeconfigPath="<KubeletKubeconfigPath>"
-if ($InstallKubeProxy) { $KubeProxyKubeconfigPath="<KubeProxyKubeconfigPath>" }
 $KubernetesHome="c:/k"
 $AntreaHome="c:/k/antrea"
-$KubeProxyLogPath="c:/var/log/kube-proxy"
 
 curl.exe -LO "https://raw.githubusercontent.com/antrea-io/antrea/${TAG}/hack/windows/Helper.psm1"
 Import-Module ./Helper.psm1
 
 Install-AntreaAgent -KubernetesVersion "$KubernetesVersion" -KubernetesHome "$KubernetesHome" -KubeConfig "$KubeConfig" -AntreaVersion "$TAG" -AntreaHome "$AntreaHome"
-New-KubeProxyServiceInterface
 
 New-DirectoryIfNotExist "${AntreaHome}/logs"
-New-DirectoryIfNotExist "${KubeProxyLogPath}"
-# Install kube-proxy service
-if ($InstallKubeProxy) { nssm install kube-proxy "${KubernetesHome}/kube-proxy.exe" "--proxy-mode=userspace --kubeconfig=${KubeProxyKubeconfigPath} --log-dir=${KubeProxyLogPath} --logtostderr=false --alsologtostderr" }
 nssm install antrea-agent "${AntreaHome}/bin/antrea-agent.exe" "--config=${AntreaHome}/etc/antrea-agent.conf --logtostderr=false --log_dir=${AntreaHome}/logs --alsologtostderr --log_file_max_size=100 --log_file_max_num=4"
 
 nssm set antrea-agent DependOnService ovs-vswitchd
-if ($InstallKubeProxy) { nssm set antrea-agent DependOnService kube-proxy ovs-vswitchd }
 nssm set antrea-agent Start SERVICE_DELAYED_AUTO_START
 
-if ($InstallKubeProxy) { Start-Service kube-proxy }
 Start-Service antrea-agent
 ```
 
-### Installation as a Pod using wins for Docker (DEPRECATED)
-
-*Dockershim was deprecated in K8s 1.20, and removed in K8s version 1.24. These
- steps may work with [cri-dockerd](https://github.com/Mirantis/cri-dockerd) but
- this is not something we validated. Antrea is no longer tested with Docker on
- Windows, and we intend to remove these steps from the documentation in Antrea
- version 2.0.*
-
-Running Antrea with Docker on Windows uses
-[wins](https://github.com/rancher/wins), which lets you run services on the
-Window hosts, while managing them as if they were Pods.
-
-#### Add Windows antrea-agent DaemonSet
-
-For example, these commands will download the antrea-agent manifest, set
-`kubeAPIServerOverride`, and deploy the antrea-agent DaemonSet when using the
-Docker container runtime:
-
-```bash
-KUBE_APISERVER=$(kubectl config view -o jsonpath='{.clusters[0].cluster.server}') && \
-curl -sL https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows.yml | \
```

Comment:
Since `antrea-windows.yml` has been removed, should we rename `antrea-windows-containerd-with-ovs.yml` to `antrea-windows-with-ovs.yml` and `antrea-windows-containerd.yml` to `antrea-windows.yml` ?

Reply:
I think given that containerd is the only supported container runtime now, this makes sense. I will make that change as part of this PR.

Reply:
Done. I added a bold note at the beginning of the document to avoid confusion, since there is a possibility that the renaming could create confusion for users. The change should be acceptable however, given that this is a v2 release.